<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="computer_name" value="usl" />
  <param name = "/use_sim_time" value="true"/>

  <node  pkg = "calibration"
         name="left_calibration_node"
         type="calibration_node_test_initialization_node"
         output="screen">
  	     <param name = "max_no_of_plane_views" value="20"/>
  	     <param name = "max_no_of_line_views" value="15"/>

  	     <param name = "usePlane" value="true"/>
	     <param name = "plane_selection_threshold" value="0.90"/>
  	     <param name = "useLines" value="false"/>

	     <param name = "generate_debug_data" value="true"/>

  	     <param name = "no_of_diff_initializations" value="1"/>
  	     <param name = "debug_data_basefilename" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/calibration/debug_data/left"/>
  	     <param name ="cam_config_file_path" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/config/nerian_passthru_faccal_left.yaml"/>
  	     <param name = "result_str" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/calibration/debug_data/C_T_L_left.txt"/>
         <param name = "initializations_file" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/matlab/initializations_left.csv"/>
  	     <param name = "results_file" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/matlab/results_left.csv"/>
	     <remap from="/points/plane" to="/velodyne_points/plane"/>
	     <remap from="/line_image1" to="/left/line_image1"/>
	     <remap from="/line_image2" to="/left/line_image2"/>
         <remap from="/line_image3" to="/left/line_image3"/>
         <remap from="/line_image4" to="/left/line_image4"/>
         <remap from="/normal1" to="/left/normal1"/>
         <remap from="/normal2" to="/left/normal2"/>
         <remap from="/normal3" to="/left/normal3"/>
         <remap from="/normal4" to="/left/normal4"/>
         <remap from="/normal_plane" to="/left/normal_plane"/>
         <remap from="/tvec_plane" to="/left/tvec_plane"/>
  </node>
</launch>
